src/snapshots/fuzz_crash/fuzz_crash_009.txt

```diff
@@ -16,14 +17,26 @@ TOKENIZE: (6:6-6:12) UnclosedString:
      ^^^^^^
 PARSER: missing_header
 ~~~TOKENS
-LowerIdent,OpenCurly,LowerIdent,Comma,Newline,CloseCurly,Newline,LowerIdent,OpAssign,Newline,StringStart,StringPart,EndOfFile
+@1-2	LowerIdent
+@2-3	OpenCurly
+@3-4	LowerIdent
+@4-5	Comma
+@5-5	Newline
+@11-12	CloseCurly
+@0-0	Newline
+@14-17	LowerIdent
+@18-19	OpAssign
+@4-4	Newline
+@25-26	StringStart
+@26-32	StringPart
+@32-32	EndOfFile
```

Comment:
Taking this as an example, I would print as:
```suggestion
LowerIdent(1:1), OpenCurly(1:2), LowerIdent(1:3), Comma(1:4),
Newline(5), ClosedCurly(2:5),
Newline(0), LowerIdent(4:1-4:3), OpAssign(4:5),
Newline(4), StringStart(6:5), StringPart(6:6-6:12), EndOfFile(6:12),
```

I think this is much easier to verify when looking at the file.

If a token is only one character, print only the start and not the end.
Print in a denser form with the token location just in parens after the token.
Print a newline before each newline token.
In the newline token, since they don't have a location, just put the identation amount.

Reply:
I honestly might opt for something like this to just be on all the time.

Reply:
Done in ce819876db..09078a643e